src/worldofzuul/Game.java

```diff
@@ -4,6 +4,11 @@
  * @author  Michael Kolling and David J. Barnes
  * @version 2006.03.30
  */
+
+/**
+ * 
+ * @author ander
+ */
 public class Game {
     private Parser parser;
     private Room currentRoom;
```

Comment:
Attributes skal også have en docblock der beskriver hvad variablen indeholder.

Reply:
fixed in: 3c24bd4bc2c2e716051d04f573df88e3e2ca974b